### ISSUE TITLE
Buffering configuration

### ISSUE THREAD

**Issue by andreasgangso** [bcc-code/bccm-player#2]
This is a good first issue.

1. Create a class in [playback_platform_pigeon.dart (see pigeons readme)](https://github.com/bcc-code/bccm-player/blob/main/pigeons), something similar to [betterplayers BufferingConfiguration], something similar to [betterplayers BufferingConfiguration](https://github.com/jhomlala/betterplayer/blob/f6a77cf6fbb515f01aa9fb459b2ee739de3e724c/docs/bufferingconfiguration)
2. Pass it as argument to [newPlayer()](https://github.com/bcc-code/bccm-player/blob/71b5b6030d547d36fe02c108b12e2a21e8429386/pigeons/playback_platform_pigeon.dart#L10C1-L10C1) because you probably can't change it for an existing exoplayer.
3. ExoPlayerController just needs to setLoadControl() with the values. Something like this: https://github.com/jhomlala/betterplayer/blob/f6a77cf6fbb515f01aa9fb459b2ee739de3e724c/android/src/main/kotlin/com/jhomlala/better_player/BetterPlayer.kt#L101

**Comment by sudhanshugairola** [comment]
any docs for using controller.gettracks and setSelectedTrack,

**Comment by andreasgangso** [comment]
yes its documented in the code so your editor should show it when you hover the methods.
You can also read the settings.dart here on github for how its used it the default flutter controls.

**Comment by sudhanshugairola** [comment]
Not default one I'm saying about list of track by controller and selection

**Comment by sudhanshugairola** [comment]
all set
one thing is missing how to get subtitles in text my controller?

**Comment by sudhanshugairola** [comment]
some time user zoom in and out or pinch to far zoom then subtitle hide on if render subtitle available the solve that issue for user and set render subtitle text over on their UI.

**Comment by sudhanshugairola** [comment]
most of user complaint on better_player that issue finally they provide for far view of subtitle over native view
